### ISSUE TITLE
LR schedule 

### ISSUE THREAD

**Issue by rom1504** [lucidrains/DALLE2-pytorch#228]
We only have fixed LR, not great

Stable diffusion use cosine schedule https://github.com/CompVis/stable-diffusion/blob/ce05de28194041e030ccfc70c635fe3707cdfc30/ldm/lr_scheduler.py#L4

Openclip too https://github.com/mlfoundations/open_clip/blob/c933765dc557d88e15be968e78d7580d95f86af8/src/training/scheduler.py

Varying LR has large impact on training

What did OpenAI dalle2 use ?

**Comment by lucidrains** [comment]
@rom1504 i think the use of cosine schedule is more rooted in tradition than evidence based, but if you can show me a paper that shows it makes a big difference, would definitely update my beliefs

i've added it [here](https://github.com/lucidrains/DALLE2-pytorch/commit/6b9b4b9e5e8f156c85d20a26a1dad8adc8713de6#diff-617450527162fa367141dbf45e8b201673573479820af0ffc56ba93b7f70947fR436)
